graylog2-web-interface/src/components/event-notifications/event-notification-types/EmailNotificationForm.jsx

```diff
@@ -50,6 +50,37 @@ Last messages accounting for this alert:
 \${end}
 `;
 
+// TODO: Default html body template should come from the server
+const DEFAULT_HTML_BODY_TEMPLATE = `<table width="100%" border="0" cellpadding="10" cellspacing="0" style="background-color:#f9f9f9;border:none;line-height:1.2"><tbody>
```

Comment:
Since we are now defaulting to an empty html body on the server side, we can actually remove that TODO,
and also remove the `DEFAULT_HTML_BODY_TEMPLATE` from `EmailEventNotificationConfig`